### PR TITLE
My portfolio desktop version

### PR DESCRIPTION
**Pull request by pratap-panabaka**
the desktop version of "My portfolio" created.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,192 +7,225 @@
...
+				</li>
+				<li class="sm"><a href=""><img src='images/m.svg' width='20' height='20' alt=''></a></li>
+				<li class="sm"><a href="https://github.com/PRATAP-KUMAR"><img src='images/github.svg' width='20'
+							height='20' alt="GitHub Profile" title=''></a></li>
+				<li class="sm"><a href=""><img src='images/v.svg' width='20' height='20' alt=''></a></li>
+
+			</ul>
+		</section>
+
+		<section class='projects-section' id="projects-section">
```
Comment (Chimwemwe-127001) [inline index.html]: - [ ] As the requirements suggest, consider replicating the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 

- Figam design
FireShot Capture 005 - ▶ Template 1 - Microverse-Student-Project-1 - www figma com

- Your Design
FireShot Capture 001 - HTML 5 My Portfolio - raw githack com

File: desktop.css [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  .get-in-touch button {
+    color: #6070ff;
+    font-weight: 500;
+    font-size: 17px;
+    background: #fff;
+    padding: 12px;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+  }
+}
```
Comment (Chimwemwe-127001) [inline desktop.css]: - [ ] I've spotted a content overlapping issue, consider taking a look at it and fixing it before your next review.

FireShot Capture 009 - HTML 5 My Portfolio - raw githack com

**Review by Chimwemwe-127001 — CHANGES_REQUESTED** [review]
# Required Changes ♻️ :warning: 
### Hi @PRATAP-KUMAR

Good job so far! 👏 🎉 

There are some issues that you still need to work on to go to the next project but you are almost there! 

so close

## Highlights
- No linter errors :heavy_check_mark:
- Repo looks professional :briefcase:
- Solid pull request :scroll:

## Required Changes ♻️

***Check the comments under the review.***

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.css [hunk 3]
```diff
@@ -83,36 +75,33 @@ body {
...
 
-.grid-box {
-  display: grid;
-  padding: 40px;
-  grid-row-gap: 88px;
+.section-2 {
+  margin: 24px;
+  border-radius: 24px;
+  background-color: #fff;
 }
```
Comment (roshan-bajgain) [inline style.css:84]: - [ ] The page is divided half of it contains the project's picture and the other half contains the project's description but I found that description took more space kindly fix it so It occupies space equally.
- [ ] Kindly fix it in all the section that content image and descritption
### your design
your
### Figma design
figma

File: style.css [hunk 4]
```diff
@@ -179,38 +174,74 @@ ul {
...
+.row-3,
+.row-4 {
+  display: flex;
+  gap: 20px;
+  background: #f7f7f9;
+  border-radius: 8px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+}
```
Comment (roshan-bajgain) [inline style.css]: - [ ] I would suggest you stick as much as possible to the Figma design's color, size, and font-family
### your design
hsssssssssssssssssss

### Figma design
fffff

File: style.css [hunk 5]
```diff
@@ -220,10 +251,10 @@ ul {
...
   align-items: center;
-  margin-top: 50px;
   border-top-left-radius: 100px;
   color: #ebebff;
-  padding: 24px;
+  padding-left: 24px;
+  padding-bottom: 24px;
 }
 
 .contact-me-text {
```
Comment (roshan-bajgain) [inline style.css:263]: - [ ] [Optional] The previous reviewer missed this so I am giving you an optional but kindly implement it. The elements of the form are centered on the page including the `Get in touch button.
- [ ]  [Optional]  Kindly increase the width of the form to make it match Figma's design.
- [ ]  [Optional]  your content is covering whole width but if you see in Figma design the content lies on middle of the page kindly fix it
### your design
yourrrrr
### Figma design
your design

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @PRATAP-KUMAR, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danielufeli — APPROVED** [review]
Hi @PRATAP-KUMAR ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- No Linter Errors✔️
- Clean Code ✔️
- Clean Design ✔️
- Descriptive Readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**